userspace/libscap/scap.c

```diff
@@ -385,8 +368,6 @@ int32_t scap_get_stats(scap_t* handle, OUT scap_stats* stats)
 	stats->n_drops_pf = 0;
 	stats->n_drops_bug = 0;
 	stats->n_preemptions = 0;
-	stats->n_suppressed = handle->m_platform->m_suppress.m_num_suppressed_evts;
```

Comment:
This is a breaking change, right? (i am ok with that, just please highlight it in the release notes!)

Reply:
Good point. I probably wanted to populate the fields from sinsp and it seemed to slip ;<

Still, even if I restore these fields, they won't be set by raw scap, so it's still technically a breaking change.

Reply:
> so it's still technically a breaking change.

Agree! I'd highlight it in the release notes and we are good to go!